### ISSUE TITLE
update submission deadline text, and create a shut-off valve

### ISSUE THREAD

**Issue by carodew** [GSA/10x#119]
## it should include …
- [x] replacing all the outdated language with current language across the site
- [x] ability to easily re-add the 'call for submissions language'

## it's done when …
- [x] it's live on the GitHub repo readme.md and merged into master
- [x] we can easily toggle the call for submissions on and off (shut off valve 🚰 )

**Comment by carodew** [comment]
I've added a shut-off valve to the site's `config.yml` file. 

It looks like this `submissions-live: false` – we'll change this variable to `true` when we open submissions again and all the buttons and related content will MAGICALLY REAPPEAR! 

Well not magically per se –– I went through the site and added if/then statements to show/hide the content based on this variable. 

I also (bonus content!) added a variable `submission-deadline` – so that when we need to update our submission deadline, we can update it there and then call that variable where ever we need to display that deadline throughout the site. 

@willcahoe I'm happy to walk you through what I did.
